test/unit/attributes.test.js

```diff
@@ -1,163 +1,233 @@
 'use strict'
 
-// TODO: convert to normal tap style.
-// Below allows use of mocha DSL with tap runner.
-require('tap').mochaGlobals()
+const tap = require('tap')
 
-const {expect} = require('chai')
 const helper = require('../lib/agent_helper')
 const {Attributes} = require('../../lib/attributes')
 const AttributeFilter = require('../../lib/config/attribute-filter')
 
 const DESTINATIONS = AttributeFilter.DESTINATIONS
 const TRANSACTION_SCOPE = 'transaction'
 
-describe('Attributes', () => {
+tap.test('#addAttribute', (t) => {
```

Comment:
Since these tests were going to serve as the start/basis of the prioritized attributes test, I converted this file to use the tap API prior to moving onto that.